.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,7 +34,7 @@ Otherwise please check these points when getting a PR done:
 - [ ] I added unit tests for my code
 - [ ] I added code comments, logging, and assertions as appropriate
 - [ ] I translated all strings visible to the user
-- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
+- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
```

Comment:
Hopefully we soon get git.permaplant.net redirection links but let us keep it like this for now.